tests/everest/entry_points/test_visualization_entry.py

```diff
@@ -1,21 +0,0 @@
-import sys
-from pathlib import Path
-from unittest.mock import patch
-
-import pytest
-
-from everest.bin.visualization_script import visualization_entry
-from everest.detached import ServerStatus
-from tests.everest.utils import capture_streams
-
-
-@patch(
-    "everest.bin.visualization_script.everserver_status",
-    return_value={"status": ServerStatus.completed},
-)
-@pytest.mark.skipif(sys.version_info.major < 3, reason="requires python3 or higher")
```

Comment:
This seems to be a test that is skipped for python < 3 which means its currently running for py 3.11

Reply:
Should just remove the skip that only applies to python 2 which is out of scope.

Reply:
fixed